tracer/src/Datadog.Tracer.Native/cor_profiler.h

```diff
@@ -203,8 +203,7 @@ class CorProfiler : public CorProfilerBase
     //
     // Tracer Integration methods #2
     //
-    long RegisterCallTargetDefinitions(WCHAR* id, CallTargetDefinition2* items, int size,
```

Comment:
I don't think you can or should delete this. I _think_ this is exactly the sort of method that we explicitly should _never_ touch 🤔

Reply:
It's never used in version mismatch scenarios